vme/src/db.cpp

```diff
@@ -1200,7 +1200,7 @@ unit_data *read_unit_string(CByteBuffer *pBuf, int type, int len, const char *wh
                     }
                 }
 
-                UNPC(u)->setDefaultPosition(pBuf->ReadU8(&g_nCorrupt));
+                UNPC(u)->setDefaultPosition(Position_int_to_enum(pBuf->ReadU8(&g_nCorrupt)));
```

Comment:
Hm. So this will change the behavior. Before, g_nCorrupt would just ++. Now it will throw an exception and I presume it might kill the process? Perhaps it is better to do something else if the parameter is OOR? E.g. set it to standing or dead. And possibly log a message?

Reply:
I need to put a lot more thought into how exceptions will be thrown/handled. I'll come back to this